src/Forms/MainForm.cs

```diff
@@ -2290,7 +2308,77 @@ public static void forceGC()
         {
             GC.Collect();
             GC.WaitForPendingFinalizers();
-            SetProcessWorkingSetSize(System.Diagnostics.Process.GetCurrentProcess().Handle, -1, -1);
+            //SetProcessWorkingSetSize(System.Diagnostics.Process.GetCurrentProcess().Handle, -1, -1);
```

Comment:
Commented out this line because it crashes on Mono in Linux.

Reply:
I was catching various bugs on Windows with 'OutOfMemory' exception, this may be the case why it happened. Will check after merge